runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -1440,6 +1455,35 @@ private void doClientReplyFlush(
             cursor.advance(partitionOffset + 1);
         }
 
+        private void doClientFlush(
+                long traceId,
+                long partitionOffset)
+        {
+            final int reserved = 0;
+            final int partitionId = group.partition.id();
+
+            assert partitionOffset >= cursor.offset : String.format("%d >= %d", partitionOffset, cursor.offset);
+
+            doFlush(sender, routeId, replyId, replySeq, replyAck, replyMax,
+                    traceId, authorization, replyBudgetId, reserved, ex -> ex
+                            .set((b, o, l) -> kafkaFlushExRW.wrap(b, o, l)
+                                    .typeId(kafkaTypeId)
+                                    .fetch(f -> f
+                                            .partition(p -> p
+                                                    .partitionId(partitionId)
+                                                    .partitionOffset(partitionOffset)
+                                                    .stableOffset(initialGroupStableOffset)
+                                                    .latestOffset(initialGroupLatestOffset)))
+                                    .build()
+                                    .sizeof()));
+
+            replySeq += reserved;
+
+            assert replyAck <= replySeq;
+
+            cursor.advance(partitionOffset + 1);
+        }
+
```

Comment:
This should be named `doClientReplyFlush` for consistency with other `doClientReply*` methods.